<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send Message</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Send Message</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="messageTB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="secretCB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Secret</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="signCB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ZIPCB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ZIP</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="R64CB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Radix-64</string>
   </property>
  </widget>
  <widget class="QComboBox" name="publicKeyComboBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="symAlgoComboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AES128</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TripleDES</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="privatekeyComboBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Public Keys</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Symmetric Algorithm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-  Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMessageButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Private key</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
